javascript/statements.json

```diff
@@ -1297,6 +1297,49 @@
               "deprecated": true
             }
           },
+          "type_css": {
+            "__compat": {
+              "description": "<code>assert {type: 'css'}</code>",
+              "support": {
+                "chrome": {
+                  "version_added": "91"
```

Comment:
CSS modules shipped in Chrome/Edge a bit after JSON modules, in 93 per https://chromestatus.com/feature/5948572598009856

Reply:
```suggestion
                  "version_added": "93"
```